deploy-tool/cert/cert.go

```diff
@@ -31,24 +36,15 @@ func IssueCertificate(deployConfig config.DeployConfig, site string, privKey *ec
 	certmagic.DefaultACME.CA = deployConfig.ACME.ACMEApiEndpoint
 
 	acme := certmagic.NewDefault()
-	if err := setStorageDriver(deployConfig, acme); err != nil {
-		return "", err
-	}
-
-	csr, err := getCSR([]string{site}, privKey)
-	if err != nil {
-		return "", fmt.Errorf("error when getting the certificate signing request: %v", err)
-	}
-
-	issued, err := acme.Issuer.Issue(context.Background(), csr)
-	if err != nil {
-		return "", fmt.Errorf("error when issuing the certificate: %v", err)
+	if err := certManager.setStorageDriver(acme); err != nil {
+		return nil, err
 	}
 
-	return string(issued.Certificate), nil
+	return acme, nil
 }
 
-func setStorageDriver(deployConfig config.DeployConfig, acme *certmagic.Config) error {
+func (certManager *CertificateManager) setStorageDriver(acme *certmagic.Config) error {
+	deployConfig := certManager.Config
```

Comment:
I think assigning a struct member to `deployConfig` and using that throughout the method obscures the fact that the value being accessed here is a member of this object, and not an argument, which I want to be obvious to the reader, especially since a different config object gets passed into the method.